### ISSUE TITLE
Erroneous text alterations on formatting

### ISSUE THREAD

**Issue by stam66** [Ferruslogic/vscode-livecodescript#13]
On VSCode 1.65.2 on MacOS Monterey:
On a particular set of text, formatting 'eats' characters. In the attached example, this affects the string **group "dates"**
Within handler text the space between words is removed. In comment text, both the space and the 'p' are removed.

![error on formatting](https://user-images.githubusercontent.com/5677273/159141754-e100f3ef-6b99-4986-a092-fba6162991c4.jpg)

**Comment by torocruzand** [comment]
Thanks @stam66.
This behavior is wrong, so we have already fixed it.
![imagen](https://user-images.githubusercontent.com/37944516/159174488-15425a0d-efe4-4c0f-9994-88b7b2cec1b9.png)

While in the case of this it is intentional.
![imagen](https://user-images.githubusercontent.com/37944516/159174583-89e80ab0-f2dc-4217-b46f-da1570f6e771.png)
Since the extension is formatting the code in a "pretty" way. If you have a chain and what precedes it is an object type, these will be joined. Like in the image. Also this formatter does the same for functions with the character ( and the [ ] ....
Eliminate extra space
...
For the beautiful format, we have looked at how other languages are formatted in vs-code beautiful.
